rails4-deps/Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y update \
     patch \
     unicorn \
     zlib1g-dev \
+    nginx \
```

Comment:
Since this is "public" image and other things might wind up using it, lets not put nginx here, but instead in the actual frontend container.